src/main/java/runner/AppRunner.java

```diff
@@ -0,0 +1,79 @@
+package runner;
+
+import client.AllInformationGenerator;
+import client.ClientLogin;
+import java.util.Scanner;
+import movie.MoviesByFactors;
+
+public class AppRunner {
+
+  static final int ALL_CINEMAS = 1;
+  static final int MORE_INFORMATION = 2;
+  static final int ALL_MOVIES = 3;
+  static final int MOVIES_BY_NAME = 4;
+  static final int MOVIES_BY_FACTORS = 5;
+  static final int BUY_TICKETS = 6;
+  static final int SEE_ALL_INFORMATION = 7;
+  static final int EXIT = 8;
+
+  public static void run() {
+
+    System.out.println("Please choose what do you want to do ");
+    System.out.println("Write 1, if you want to see a list of all cinemas   ");
+    System.out.println("Write 2, if you want to see a list of movies in some cinema   ");
+    System.out.println("Write 3, if you want to see a list of all movies ");
+    System.out.println("Write 4, if you want to see a more information about movie by name ");
+    System.out.println("Write 5, if you want to see all the movies by various factors");
+    System.out.println("Write 6, if you want to buy or add in booking the ticket");
+    System.out.println("Write 7, if you want to see the information  about all your bookings");
+    System.out.println("Write 8, if you want exit");
+
+    Scanner scan = new Scanner(System.in);
+    int input = scan.nextInt();
+
+    while (input != EXIT) {
+      switch (input) {
+
+        case ALL_CINEMAS:
+          System.out.println("These are all our cinemas: ");
+          AllCinemas.showAllCinemas();
```

Comment:
No validation for bad inputs. Any mistakes crash your app.

Reply:
First